api-server/src/venues/dto/create-update-venue-request.ts

```diff
@@ -13,6 +14,26 @@ export class CreateVenueRequest {
     @ApiProperty({example: 'https://maps.google.com/maps/foo/bar'})
     @IsOptional()
     g_map_link?: string;
+
+    @Column
+    @ApiProperty({example: 32.7 })
+    @IsOptional()
+    gps_lat?: number
+
+    @Column
+    @ApiProperty({example: 76.3 })
+    @IsOptional()
+    gps_long?: number
+
+    @Column
+    @ApiProperty({example: 17.1 })
+    @IsOptional()
+    gps_alt?: number
+
+    @Column
+    @ApiProperty({ example: false })
+    @IsOptional()
+    is_soft_deleted: boolean;
```

Comment:
Is there a reason the create request would include the soft delete flag?

Reply:
Hmm probably not ideal, should just get defaulted to false on create. I wonder if that was copy/paste mistake on my part or if this dto is overloaded. I'll poke at this  a bit

Reply:
yeah not sure why I added, that, fixing now